src/components/delete-track-dialog.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+    Box,
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    FormControl,
+    FormHelperText,
+    TextField,
+} from "@material-ui/core";
```

Comment:
make sure to remove all unnecessary imports

Reply:
i'll get back to that when the PR is done